components/eamxx/src/share/tests/field_tests.cpp

```diff
@@ -847,6 +847,18 @@ TEST_CASE ("update") {
     f2.scale(2.0);
     REQUIRE (views_are_equal(f1, f2));
   }
+
+  SECTION ("scale_inv") {
```

Comment:
I guess we could add a test for type = int using 4 and 2 or something like that, which would have caught my previous comment.